tests/regression/lp-1898038/test-multipass-support-app/bin/test-multipass-support

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+cat <<EOF > /tmp/test-profile
+#include <tunables/global>
+
+profile test-profile {
+  #include <abstractions/base>
+
+}
+EOF
+/usr/sbin/apparmor_parser -r /tmp/test-profile
```

Comment:
Nitpick: It'd be great to `apparmor_parser --remove` this profile in test cleanup. It might be easier to just have the profile as a real file and use a an app `.sh` which just `exec /bin/sh "$@"`. This way you can write tests that run arbitrary command straight in the `task.yaml`. In addition you can easily remove the profile from memory, since you have access to the profile source with the profile name.

Reply:
Thanks - I just added this in abf0f628b0